.github/workflows/appimage.yml

```
@@ -202,45 +206,54 @@ jobs:
+          git config --global safe.directory '*'
           git checkout -b "release/${SEMVER_STR}"
       - name: Update build file
         run: |
           sed -i -e "s|^pkgver=.*$|pkgver=${SEMVER_STR}|" PKGBUILD
           sed -i -e 's|^pkgrel=.*$|pkgrel=1|' PKGBUILD
-          sed -i -e "s|^sha256sums_x86_64=.*$|sha256sums_x86_64=('${AUR_SHA256_X64}'|" PKGBUILD
-          sed -i -e "s|^sha256sums_aarch64=.*$|sha256sums_aarch64=('${AUR_SHA256_AARCH64}'|" PKGBUILD
-          makepkg --printsrcinfo > .SRCINFO
+          updpkgsums && makepkg --printsrcinfo > .SRCINFO
         env:
           AUR_SHA256_X64: ${{ steps.checksums.outputs.x64-sha256sum }}
```

actionlint (1.7.11)

[error] property "checksums" is not defined in object type {}

(expression)